lib/util.js

```diff
@@ -575,13 +575,9 @@ function formatProperty(ctx, value, recurseTimes, visibleKeys, key, array) {
       }
       if (str.indexOf('\n') > -1) {
         if (array) {
-          str = str.split('\n').map(function(line) {
-            return '  ' + line;
-          }).join('\n').substr(2);
+          str = str.replace(/\n/g,'\n  ');
```

Comment:
minor linting issue... need to put a space after the comma in param lists. I'll fix that when landing tho.